extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/shared.properties.js

```diff
@@ -31,16 +29,14 @@ angular.module('PaperUI.controllers').service('sharedProperties', function() {
             var modArr = this.getModuleArray(type);
 
             if (!value.id) {
-                value.id = type + "_" + tId;
+                value.id = type.substring(0, 1) + "_" + Date.now();
```

Comment:
But it uses the max number in the respective module and not in the rule overall. Is that fine?